### DISCUSSION TITLE
Setting up with seedbox/cloud server?

### DISCUSSION THREAD

**Discussion by ElysiumFic** [advplyr/audiobookshelf#878] [Q&A]
I've been using a seedbox with Plex + Prologue for my audiobooks, but I've never been thrilled with it (Prologue is great, but Plex just isn't meant for audiobooks, even with plugins.)

The reason I've been using a seedbox is because I've had a hard drive die on me and lost my whole (nearly 1TB) audiobook collection (hence the reason I'm re-organizing, now that I've gotten most of them together again from various sources.) Also, I like the ability to stream without having to actually download the audiobook.

I just learned about this option, and since I'm in the middle of re-organizing my library, I'd like to give it a try. However, my knowledge base is mostly just user-level stuff, sometimes higher if I've got very clear instructions and everything works the way the instructions say it should. Setting this up might be beyond my skillset, but I'm willing to give it a try.

However, I'd like to install to my seedbox (and hopefully be able to stream the same way I have been, though that's not mandatory.) I don't have root/sudo access, but the host does have [generic instructions](https://www.feralhosting.com/wiki/slots/generic-install-guide), as well as instructions for [compiling from source](https://www.feralhosting.com/wiki/slots/install-source) and [extracting from .deb files](https://www.feralhosting.com/wiki/slots/install-deb).

The host also allows ffmpeg on the servers.

So on the surface, it looks somewhat feasible that I can do this, but I'm just wondering if anyone has tried yet, and if so, is there anything I should beware of that will prevent things from working the way it seems they should? I don't want to spend possibly days knocking my head against a brick wall trying to figure out how to make it work, only to find out it never would have worked from the start.

**Comment by advplyr** [comment]
I have no experience with this. It seems like the best option is extracting the deb file but I don't understand those instructions.

Here is a path to the latest deb file:
`https://advplyr.github.io/audiobookshelf-ppa/audiobookshelf_2.1.2_amd64.deb`

You may also have luck asking in the discord server

**Comment by Guruleenyc** [comment]
Consider unraid for drive fault tolerance.
